src/app/frontend/deploy/deploy_controller.js

```diff
@@ -80,44 +57,22 @@ export default class DeployController {
     /** @private {!md.$dialog} */
     this.mdDialog_ = $mdDialog;
 
-    /** @private {boolean} */
-    this.isDeployInProgress_ = false;
+    /** @export {boolean}*/
+    this.isDeployInProgress = false;
+
+    /** @export {!Object}*/
```

Comment:
The type of this should be "an object that has deploy() function". Right? This can be written as `{{deploy:function()}}`. Also, add a quick comment saying what this var is.